src/pages/DataAccessRequestApplication.js

```diff
@@ -481,43 +556,87 @@ class DataAccessRequestApplication extends Component {
     this.setState({ showDialogSave: true });
   };
 
-  dialogHandlerSubmit = (answer) => (e) => {
+  //Can't do uploads in parallel since endpoints are post and they both alter attributes in json column
+  //If done in parallel, updated attribute of one document will be overwritten by the outdated value on the other
```

Comment:
I had to reckon with this as well with DAC membership updates. We have a utility function, `utils/PromiseSerial` that will force a list of async functions to execute serially. Probably overkill for what you're doing here, but might be helpful in the future.